### ISSUE TITLE
What about a "new version" notifier ? 

### ISSUE THREAD

**Issue by dridk** [labsquare/fastQt#63]
Would you like to check for new version when application start ?

**Comment by ikit** [comment]
yes, but in my opinion, it shall be invisible if nothing new or no internet connexion.

**Comment by Arkanosis** [comment]
Not if installed using the distro's package manager — that would be more of an inconvenience. Definitely if it has been installed manually, otherwise people manage to keep using an outdated version for years (I'm always amazed when I see that some people are still using OpenOffice). This shall be done the Firefox way (small non-modal notification bar) and not the Notepad++ way (intrusive modal popup), because the latter sucks big time.
